tests/epyccel/test_epyccel_functions.py

```diff
@@ -1,18 +1,11 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring/
 # coding: utf-8
 
-import shutil
 import pytest
 import numpy as np
 
 from pyccel.epyccel import epyccel
 from pyccel.decorators import types
-from conftest       import *
-
-
-def clean_test():
-    shutil.rmtree('__pycache__', ignore_errors=True)
```

Comment:
I suspect if we use `ignore_errors = True` in conftest this may fix the windows problems. Hopefully

Reply:
'''    
 if f in  ("__pyccel__", "__epyccel__"):
            shutil.rmtree( file_name )                              here ?  <----------------

'''

Reply:
yes